src/renderers/shaders/ShaderChunk/skinning_vertex.glsl

```diff
@@ -7,6 +7,7 @@
 	skinned += boneMatY * skinVertex * skinWeight.y;
 	skinned += boneMatZ * skinVertex * skinWeight.z;
 	skinned += boneMatW * skinVertex * skinWeight.w;
-	skinned  = bindMatrixInverse * skinned;
+
+	transformed = ( bindMatrixInverse * skinned ).xyz;
```

Comment:
I am making the assumption that the transformations applied to `skinned` are affine. If that's the case, then you can drop the w-component since it will always be 1. The matrices in question _should_ be affine, since they originate from the Object3D's matrixWorld property, and are all intended to rotate/scale/skew/translate vertices around, rather than to project vertices.

The matrices in question:
https://github.com/mrdoob/three.js/blob/r85/src/objects/Skeleton.js#L136
https://github.com/mrdoob/three.js/blob/r85/src/objects/SkinnedMesh.js#L108

Is there a reason to suspect these matrices may sometimes not be affine?